packages/core/src/plugins/minimize.ts

```diff
@@ -64,25 +64,20 @@ export const parseMinifyOptions = (
   jsOptions?: SwcJsMinimizerRspackPluginOptions;
   cssOptions?: LightningCssMinimizerRspackPluginOptions;
 } => {
-  const { minify } = config.output;
-
-  if (minify === false || !isProd) {
-    return {
-      minifyJs: false,
-      minifyCss: false,
-    };
+  if (!isProd) {
+    return { minifyJs: false, minifyCss: false };
   }
+  const { minify: _minify } = config.output;
+  const minify =
+    _minify && typeof _minify === 'object'
```

Comment:
I prefer to use the former implementation because it seems simpler. 😂

Reply:
Done!